pages/download.tsx

```diff
@@ -20,19 +20,34 @@ export default function Download(): ReactElement {
   const headingClasses = 'text-5xl font-semibold';
   const subtitleClasses = classNames('text-2xl', 'lg:text-3xl');
   const linkContainerClasses = classNames(
-    'flex flex-wrap justify-center pb-12',
-    'lg:items-center'
-  );
-  const downloadContainerClasses = classNames(
-    'pb-2 lg:py-3 lg:border-r lg:border-dashed z-10'
+    'grid grid-cols-2 gap-2 mb-4',
+    'lg:w-max',
+    'xl:w-full'
   );
```

Comment:
Would be good to constrain the container, on wide screens the items in each row drift further apart.

Reply:
Fixed with mx-auto.